Source/Private/ASMultidimensionalArrayUtils.mm

```diff
@@ -226,6 +226,7 @@ void ASMoveElementInTwoDimensionalArray(NSMutableArray *mutableArray, NSIndexPat
 
 id ASGetElementInTwoDimensionalArray(NSArray *array, NSIndexPath *indexPath)
 {
+  ASDisplayNodeCAssertNotNil(indexPath, @"Expected non-nil index path");
```

Comment:
Previously a nil index path will cause the next assert to fail with an unclear message: `'Expected index path of length 2. Index path: (null)'`. Adding this assertion to correctly reflect what's wrong.